README.md

```diff
@@ -58,19 +58,18 @@ OR
 
     docker exec -it frontera_prtl_django /bin/bash
     python3 manage.py migrate
-    python3 manage.py createsuperuser
-    python3 manage.py collectstatic
+    python3 manage.py collectstatic --noinput
 
 #### Initialize the CMS in the `frontera_prtl_cms` container:
 First, copy the sample secrets:
 
-    cp /server/conf/cms/secrets.sample.py /server/conf/cms/secrets.py
+    cp server/conf/cms/secrets.sample.py server/conf/cms/secrets.py
+
 Then, run migrations and `collectstatic`:
 ```
     docker exec -it frontera_prtl_cms /bin/bash
     python3 manage.py migrate
-    python3 manage.py createsuperuser
```

Comment:
I actually only seem to need to run `createsuperuser` for `frontera_prtl_cms`, since I can't ever seem to login with my normal tacc account there. Are there steps for that?

Reply:
I *think* i can login with my normal tacc account but let me check before merging.

Reply:
Oh… yeah. I'm with Sal on this. I recently tested logging into CMS with TACC account. I could not do it either. So, I suggest retaining this command. One easy way to resolves this is:

```
python3 manage.py createsuperuser # Unless you will only login with your TACC account
```

> P.S. The [code for LDAP support in CMS](https://github.com/TACC/CORE-cms/commit/1c326b911f2874ae76b53840bf02f061742b09cd#diff-16fd09f645fd22d5389d1711d21583bb9ce289bb09179cb8fa2b6fbf41aac385R20-R34) is more than a year old.

Reply:
I can't get into CMS with my TACC account either.  

I'll revert the removal of 'python3 manage.py createsuperuser' for CMS.  but I like your suggestion of adding `# Unless you will only login with your TACC account`.  

@rstijerina @tacc-wbomar , what about using that suggestion for just the django container:
```
    docker exec -it frontera_prtl_django /bin/bash
    python3 manage.py migrate
    python3 manage.py collectstatic --noinput
    python3 manage.py createsuperuser # Unless you will only login with your TACC account
```
and leaving the other as is:
```
    docker exec -it frontera_prtl_cms /bin/bash
    python3 manage.py migrate
    python3 manage.py collectstatic --noinput
    python3 manage.py createsuperuser
```

Reply:
👍

Reply:
Okie. I'll create a ticket for the CMS LDAP login issue.

Reply:
@xandroc Do you know how to sign in via LDAP to the cms component?

Reply:
> @xandroc Do you know how to sign in via LDAP to the cms component?

@rstijerina @nathanfranklin @tacc-wbomar 
If you're behind the firewall, you may be authenticating just fine, but the CMS admin site won't let users login to /admin unless they have the right access, this means we need to create a superuser account with the same username we want to use for login. In this case, because creating a super user assumes a new account is being created (and authing with ldap imports the user account from ldap even if login failed) you may have to update the super user flag in the db for the account being used to log in

Reply:
> I actually only seem to need to run `createsuperuser` for `frontera_prtl_cms`, since I can't ever seem to login with my normal tacc account there. Are there steps for that?

this is expected behavior, even if ldap auth is successful the /admin site won't let you in unless you have access. This is good because it means that not just anyone with TACC credentials can get access to /admin. Here's some more detail, https://gitlab.tacc.utexas.edu/wma-cms/frontera-cms/blob/master/README.md#L64